srv/isc-agent/plugins/tcp/http/iscagentsubmit.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from twisted.internet import reactor
+from twisted.web.client import Agent, FileBodyProducer
+from twisted.web.http_headers import Headers
+
+from plugins.tcp.http import models
+
+agent = Agent(reactor)
+logger = logging.getLogger(__name__)
+logs = models.logs
+body = FileBodyProducer(logs)
+
+d = agent.request(
+    b"POST",
+    b"127.0.0.1:8000",
+    Headers({'content-type': 'application/json', 'User-Agent': 'DShield PyLib 0.1'}),
+    body,
+)
+
+
+def post_response(ignored):
+    print('Data posted')
+    logger.warning('Data posted')
+
+
+def post_Shutdown(ignored):
+    reactor.stop()
+
+
+def post():
```

Comment:
For the posting logic i think there should be a flag that allows the user to determine wether or not the logs should be sent back to ISC. Itll be on by default but should allow the user to pass a value to turn it off